<launch>


	<arg name="rviz" default="true" />
	<arg name="time_file" default=""/>	
	<arg name="out_path" default=""/>	
	<arg name="out_origin_path" default=""/>
	<arg name="pose_log" default="false"/>
	<arg name="pose_log_file" default=""/>
	<arg name="cluster_out_file" default=""/>
	<arg name="time_breakdown_file" default=""/>

	<rosparam command="load" file="$(find m_detector)/config/avia/avia0.yaml" />
	<param name="dyn_obj/out_file" type="string" value="$(arg out_path)" />
	<param name="dyn_obj/out_file_origin" type="string" value="$(arg out_origin_path)" />
	<param name="dyn_obj/time_file" type="string" value="$(arg time_file)" />
	<param name="dyn_obj/pose_log" type="bool" value="$(arg pose_log)" />
	<param name="dyn_obj/pose_log_file" type="string" value="$(arg pose_log_file)" />
	<param name="dyn_obj/cluster_out_file" type="string" value="$(arg cluster_out_file)" />
	<param name="dyn_obj/time_breakdown_file" type="string" value="$(arg time_breakdown_file)" />

	<node pkg="m_detector" type="dynfilter" name="dynfilter" output="screen" />
	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find m_detector)/rviz/demo.rviz" />
	</group>

  	  

</launch>
